src/doc/src/reference/workspaces.md

```diff
@@ -49,6 +52,7 @@ where the workspace's `Cargo.toml` is located.
 
 ```toml
 [workspace]
+resolver = "2"
```

Comment:
Note: for this case, it isn't needed because it will be inferred from `package.edition`

Reply:
noted :+1:  I actually mostly used the virtual ones

Reply:
This entry with a non-virtual workspace seems to still be here

Reply:
sorry, no idea how I missed that, removed